REQUIRE

```diff
@@ -1,4 +1,4 @@
 julia 0.4
-Compat 0.7.14
+Compat 0.7.14+
```

Comment:
Correct. I'm just not sure what's the status of https://github.com/JuliaLang/Compat.jl/pull/192 .